apps/site/src/pages/api/verify-email.api.ts

```diff
@@ -78,7 +88,28 @@ export default createBaseHandler<
         );
       }
 
-      await user.update(db, { hasVerifiedEmail: true });
+      if (emailVerificationCode.variant === "linkWordpress") {
+        const { wordpressInstanceUrl } = emailVerificationCode;
+        if (!wordpressInstanceUrl) {
+          return res.status(500).json(
+            formatErrors({
+              msg: "Internal error. Please try again.",
+              param: "code",
+              value: code,
+            }),
+          );
+        }
+
+        await emailVerificationCode.setToUsed(db);
+        await user.addWordpressInstanceUrlAndVerify(
+          db,
+          wordpressInstanceUrl,
+          !user.hasVerifiedEmail,
```

Comment:
Similarly to the comment above, because of the exception thrown `if (user.hasVerifiedEmail)`, can this referrer value ever be `false`?